table/sst_file_dumper.cc

```diff
@@ -456,10 +457,20 @@ Status SstFileDumper::ReadSequential(bool print_kv, uint64_t read_num,
       read_options_, moptions_.prefix_extractor.get(),
       /*arena=*/nullptr, /*skip_filters=*/false,
       TableReaderCaller::kSSTDumpTool);
+
+  const Comparator* ucmp = internal_comparator_.user_comparator();
+  size_t ts_sz = ucmp->timestamp_size();
+
+  Slice from_slice = from_key;
+  Slice end_slice = to_key;
+  std::string from_key_buf, end_key_buf;
+  auto [from, end] = MaybeAddTimestampsToRange(
+      has_from ? &from_slice : nullptr, has_to ? &end_slice : nullptr, ts_sz,
+      &from_key_buf, &end_key_buf);
```

Comment:
It might make sense to use `to` instead of `end` here to be consistent with the command line parameter names.

Reply:
Cool, it's indeed unnecessary and distracting renaming.